src/repo/build.ocp

```diff
@@ -7,6 +7,7 @@ begin library "learnocaml-repository"
   requires = [
     "ocplib-json-typed"
     "xor"
+    "omd"
```

Comment:
I am surprised that this dependency was not already here since `src/repo/learnocaml_parser.ml` uses `Omd_backend`. I must have missed something...

Reply:
It's because the compilation of `learnocaml_tutorial_parser` is handled in an other stanza, line 16. And this stanza had this dependency specified.

Reply:
OK, thanks.